support/cas-server-support-yubikey-core/src/main/java/org/apereo/cas/adaptors/yubikey/registry/BaseYubiKeyAccountRegistry.java

```diff
@@ -1,23 +1,42 @@
 package org.apereo.cas.adaptors.yubikey.registry;
 
-import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
 import lombok.extern.slf4j.Slf4j;
+import org.apereo.cas.CipherExecutor;
 import org.apereo.cas.adaptors.yubikey.YubiKeyAccountRegistry;
 import org.apereo.cas.adaptors.yubikey.YubiKeyAccountValidator;
-import lombok.ToString;
+
+import java.io.Serializable;
 
 /**
  * This is {@link BaseYubiKeyAccountRegistry}.
  *
  * @author Misagh Moayyed
+ * @author Dmitriy Kopylenko
+ *
  * @since 5.2.0
  */
 @Slf4j
 @ToString
 @NoArgsConstructor(force = true)
-@AllArgsConstructor
+@Getter
+@Setter
 public abstract class BaseYubiKeyAccountRegistry implements YubiKeyAccountRegistry {
-    /** Account validator. */
+
+    /**
+     * CipherExecutor.
+     */
+    private CipherExecutor<Serializable, String> cipherExecutor;
+
+    /**
+     * Account validator.
+     */
     protected final YubiKeyAccountValidator accountValidator;
+
+    public BaseYubiKeyAccountRegistry(final YubiKeyAccountValidator accountValidator) {
```

Comment:
Please remove, and use @RequiredArgsConstructor instead. That would only make a constructor available for `final` fields ignoring others, and you already have the field.